pom.xml

```diff
@@ -20,6 +20,20 @@
             <version>5.9.0</version>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>4.6.1</version>
```

Comment:
Please refer to comments made on Luis' changes, you can work together on the changes for using properties to define the version of the dependencies, I talked about this during today's session (4 PM)

https://github.com/ViktoriyaRyazhska/Gdl-01.Java/pull/23/files#r983919021